packages/codemirror/src/factory.ts

```diff
@@ -25,8 +25,8 @@ export class CodeMirrorEditorFactory implements IEditorFactoryService {
         'Cmd-Left': 'goLineLeft',
         Tab: 'indentMoreOrinsertTab',
         'Shift-Tab': 'indentLess',
-        'Cmd-/': 'toggleComment',
-        'Ctrl-/': 'toggleComment',
+        'Cmd-/': (cm) => cm.toggleComment({indent:true}),
```

Comment:
I ran the linter locally, and these lines should be formatted as `cm => cm.toggleComment({ indent: true }),`, mind updating them?